src/payment/models/history/history.service.ts

```diff
@@ -464,9 +464,7 @@ export class HistoryService {
   }
 
   private getAssetSymbol(dexName: string): string {
-    // TODO: new col in asset table to differentiate stocks and crypto token?
-    dexName = dexName.replace('-Token', '');
-
+    // TODO: use col from asset table to differentiate stocks and crypto token?
```

Comment:
is this comment still relevant or can it be removed?

Reply:
It's still relevant. Instead of the asset list below, we could fetch the asset from database to get the category.